### DISCUSSION TITLE
Rogue Scheduler running after crash / restart

### DISCUSSION THREAD

**Discussion by jmwestbe** [quartznet/quartznet#643] [General] [answered]
### Issue
We are using the scheduler as a component of a Windows Service to create scheduled jobs.
When the service crashed and was restarted, all scheduled jobs are now duplicated - implying that the scheduler instance survived the crash (and we have lost a handle on it).  We have since hard-restarted the machine and the 'zombie' scheduler continues to run in addition to the one used by the currently running windows service.  Note that the scheduler *should* be using the default `RAMJobStore` and all jobs use the default attribute of `RequestsRecovery = false`

### Information Request
While I would love to know how this happened - and why in the world a restart doesn't kill the rogue process - I am more interested in knowing how I can get a handle on the underlying process that the rogue scheduler might be running on so that I can kill it.  Any suggestions on where to look to find that?

**Comment by lahma** [comment, marked as answer]
Quartz and its thread are part of the hosting process so when that process (or the entire operating system) goes down, so should are Quartz processing too. I suggest that you add logging and invetigate how the scheduler starts and what decisions it makes.

**Comment by jmwestbe** [comment]
Will do - thank you for confirming that it is tied to the host process.
